mappings/net/minecraft/text/OrderedText.mapping

```diff
@@ -0,0 +1,32 @@
+CLASS net/minecraft/class_5481 net/minecraft/text/OrderedText
```

Comment:
The `mapper` parameters are a bit unclear on the non-`map` methods. Maybe `codePointMapper` would be better since it maps code points?